examples/c/src/common.cpp

```diff
@@ -39,46 +39,11 @@ void Timing::Log(const int prompt_tokens_length, const int new_tokens_length) {
   std::cout << "-------------" << std::endl;
 }
 
-void TerminateSession::signalHandler(int signum) {
```

Comment:
DO we not need an example for terminate session?

Reply:
The termination feature is still shown in the example. But it is implemented differently now. We don't explicitly spawn a background thread to terminate generation. We do it through control+c signal.